roqoqo-iqm/src/interface.rs

```diff
@@ -124,7 +141,17 @@ pub fn call_circuit<'a>(
                         if let CalculatorFloat::Str(s) = meas_readout {
                             if s == &readout {
                                 found = true;
-                                i.qubits.push(_convert_qubit_name_qoqo_to_iqm(*o.qubit()));
+                                let iqm_qubit = _convert_qubit_name_qoqo_to_iqm(*o.qubit());
+                                if !i.qubits.contains(&iqm_qubit) {
+                                    i.qubits.push(iqm_qubit);
```

Comment:
This may just be personal preference, but I find that `i` is usually used to iterate through a range of integers, and here it's an operation. I would personally change `i` to `op`

Reply:
I have already used bot `o` and `op` in the same cycle, to refer to qoqo operations. I had picked `i` because we are iterating over IQM `instructions`, but if you have a better idea for the name I'll change it

Reply:
how about `instr`?